src/gridstack.ts

```diff
@@ -2181,7 +2181,7 @@ export class GridStack {
       if (node._temporaryRemoved) return; // handled by dropover
       let distance = ui.position.top - node._prevYPix;
       node._prevYPix = ui.position.top;
-      if (!this.opts.disableAutoScroll) {
+      if (this.opts.draggable.scroll === false) {

```

Comment:
you mean
if (this.opts.draggable.scroll !== false) {...}

you also should update types.ts
gridDefaults: GridStackOptions = {...
draggable: { handle: '.grid-stack-item-content', appendTo: 'body', scroll: true },

and update the doc mentioning that new option...

Reply:
Done 😄